dev/com.ibm.ws.security.social/resources/com/ibm/ws/security/social/resources/SocialMessages.nlsprops

```diff
@@ -481,8 +491,8 @@ RESPONSE_STATUS_MISSING_OR_ERROR=CWWKS5477E: The response status cannot be found
 RESPONSE_STATUS_MISSING_OR_ERROR.explanation=The request likely failed because it was unauthorized or missing information such as a required request parameter or header. If the response status was not null, the response status code and reason is included in the message.
 RESPONSE_STATUS_MISSING_OR_ERROR.useraction=Check the response status code in the message, if present. Verify that all required parameters and headers were included in the request.
 
-# 0=URL to which the request was submitted, 1=Error message
-RESPONSE_STATUS_UNSUCCESSFUL=CWWKS5478E: The request to the endpoint [{0}] failed. The error was: {1}
+# 0=URL to which the request was submitted, 1=HTTP response status (e.g. 404 Not Found), 2=Error message
+RESPONSE_STATUS_UNSUCCESSFUL=CWWKS5478E: The request to the endpoint [{0}] failed. The response status is [{1}]. The error is: {2}
```

Comment:
Change
"The request to the endpoint [{0}] failed."
To
"The request to the [{0}] endpoint failed."